components/Starknet/modules/architecture_and_concepts/pages/Accounts/deploying_new_accounts.adoc

```diff
@@ -1,61 +1,74 @@
 [id="deploying_new_accounts"]
-= Deploying new accounts
+= Deploying a new account
 
-Starknet provides the `deploy_account` transaction in order to deploy new accounts to the
-network.
+The `DEPLOY_ACCOUNT` transaction deploys new accounts to the network.
 
-== Deploy account transaction
+Upon receiving a `DEPLOY_ACCOUNT` transaction, the sequencer performs the following steps:
 
-In order to deploy a new account on StarkNet you need to complete the following steps:
+. Verifies that the address has funds to pay for the deployment.
+. Executes the constructor with the given arguments.
+. Executes the `+__validate_deploy__+` entry point. For more information, see xref:#validate_deploy[].
+. Charges fees from the new account address.
+. Sets the account's nonce to `1`.
 
-* Decide on the account contract that you want to deploy
-* Compute your would-be account address off-chain
-* Send funds to this address
+.Procedure
 
-Once the address has enough funds to pay for the deployment, you can then send a `deploy_account` transaction.
+. For the account contract that you want to deploy, compute the offchain account address.
+// How do I do that?
+. Send funds to this address.
+// How do I do that?
+. When the address has enough funds to pay for the deployment, send a `DEPLOY_ACCOUNT` transaction.
+// How do I do that?
 
-== Transaction flow
+After the deployment has completed successfully, it is recommended to validate the account deployment.
 
-Upon receiving a `deploy_account` transaction, the sequencer will:
+[#DEPLOY_ACCOUNT_restrictions]
+== `DEPLOY_ACCOUNT` constructor restrictions
```

Comment:
These restrictions apply uniformly to:
1. validate
2. validate_deploy
3. validate_declare
4. constructor, when ran in a DEPLOY_ACCOUNT txn (i.e. if an account is deployed from an existing via the deploy syscall, the constructor can run differently)

Hence, these restrictions should be under general validate limitations section. This new section should start with the motivation of DOS protection around validate, which is atm somewhere below.

Reply:
Done